lib/models/ObjectMD.ts

```diff
@@ -183,7 +183,7 @@ export default class ObjectMD {
             'content-length': 0,
             'content-type': '',
             'content-md5': '',
-            'content-language': '',
+            'content-language': undefined,
```

Comment:
is it worth keeping these (undefiend) fields in the default value?
since:
- the model is defined above (so does not help with 'documenting')
- undefined will not be serialized
- when this type is used, methods should be used to access the data and handle the missing optional fields